webportal/src/app/create_item/components/create_completed.jsx

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Stack, Text, Link } from 'office-ui-fabric-react';
+import { FontIcon } from 'office-ui-fabric-react/lib/Icon';
+import {
+  mergeStyles,
+  mergeStyleSets,
+} from 'office-ui-fabric-react/lib/Styling';
+import { TYPE_ENUM } from 'App/utils/constants';
+
+const iconClass = mergeStyles({
```

Comment:
why use mergeStyles?

Reply:
I use the example code here [Icon with custom color](https://developer.microsoft.com/en-us/fluentui#/controls/web/icon)